instagram-page-react/src/pages/profilePage/reducers/selectors/profilePage.posts.js

```diff
@@ -1,16 +1,30 @@
 import { compose } from 'recompose';
+import { defaultMemoize } from 'reselect';
 
 // lodash
 import _property from 'lodash/property';
 
+// utils
+import reverseArray from "../../../../utils/reverseArray";
+
 const getProfilePageState = _property('profilePage');
 
 const getPostListState = compose(
     _property('posts'),
     getProfilePageState
 );
 
-export const getPosts = compose(
+const getPostsList = compose(
     _property('data'),
     getPostListState
 );
+
+const reversePosts = defaultMemoize(reverseArray);
```

Comment:
First, naming this as `reversePosts` couples it with posts, we should instead call it `memoizedReverseArray`

Second, we are memoizing at the file level, will this cause an issue in case another selector starts using this same function?